File: Important Coding Problems/Math/MissingNumber.java
import java.util.Scanner;

class MissingNumber {
    public static void main(String[] args) {
        int n = 8;
        int[] arr = { 1, 2, 3, 4, 5, 6, 8 };
        int sum = 0;
        for (int i = 0; i < arr.length; i++) {
            sum += arr[i];
        }
        int total = (n * (n + 1)) / 2;
        System.out.println(total - sum);
    }
}

File: CSES/missing_number.cpp
#include <iostream>
#include <vector>

using namespace std;

int main() {
    long long n;
    cin >> n;
    vector<long long> arr(n - 1);

    for (long long i = 0; i < n - 1; i++) {
        cin >> arr[i];
    }

    long long total = n * (n + 1) / 2;
    long long sum = 0;
    for (long long num : arr) sum += num;

    cout << total - sum << endl;
    return 0;
}
